README.md

```diff
@@ -275,6 +279,41 @@ test. The Terraform files then allow us to build what the resource structure is
 expected to look like and once the tool parses the API response, we can compare
 that to the static file.
 
+Suggested local testing steps:
+
+1. Create a main file (do not commit this up)
+
+      ```bash
```

Comment:
can you outdent these please? we don't need indentation + code blocks in markdown so let's keep it consistent with the existing format of outdented + code fences.